services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None,
+) -> QuerySet:
+    match (genres_ids, actors_ids):
+        case (None, None):
+            return Movie.objects.all()
+        case (_, None):
+            return Movie.objects.filter(genres__id__in=genres_ids).distinct()
+        case (None, _):
+            return Movie.objects.filter(actors__id__in=actors_ids).distinct()
+        case _:
+            return Movie.objects.filter(
+                genres__id__in=genres_ids, actors__id__in=actors_ids
+            ).distinct()
```

Comment:
It is not a clear solution. Simplify this code. You must have only two if statements:
1. When genres_ids is not None
2. When actors_ids is not None